gptresearcher/context/llm_utils.py

```diff
@@ -98,7 +98,7 @@ async def stream_response(model, messages, temperature, max_tokens, websocket):
     return response
 
 
-def choose_agent(task: str) -> dict:
+def choose_agent(researcher, task: str) -> dict:
```

Comment:
@gregdrizz same here. please remove all researcher objects from generic functions. Pass the relevant params such as "model"